src/utils/read-lines-separated-file.js

```diff
@@ -0,0 +1,13 @@
+import { readFile } from 'fs';
+
+export default function readLinesSeaparatedFile(filePath) {
+  return new Promise((resolve, reject) => {
+    readFile(filePath, 'utf-8', (err, content) => {
+      if (err) {
+        return reject(err);
+      }
+
+      return resolve(content.split('\n'));
```

Comment:
Could we use a regular expression here, on windows a new line is `\r\n`.

Reply:
I was under the impression fs.readLine always interprets `\r\n` as simply `\n` but now I couldn't find anything about it. Fixed.